documentation/src/docs/deployment/kubernetes-install.md

```diff
@@ -31,8 +31,9 @@ You may obtain a copy of the License at
 
 ## Requirements
 
-- Kubernetes 1.8+
-- An existing Apache Zookeeper 3.5 cluster. This can be easily deployed using our [Zookeeper Operator](https://github.com/pravega/zookeeper-operator).
+- Kubernetes 1.15+
+- An existing Apache Zookeeper 3.6.1 cluster. This can be easily deployed using our [Zookeeper Operator](https://github.com/pravega/zookeeper-operator).
+- Bookkeeper setup using [bookkeeper-operator](https://github.com/pravega/bookkeeper-operator)
```

Comment:
Please, be consistent with the formatting: If previously you used `Zookeeper Operator`, now the right formatting would be `Bookkeeper Operator` (not `bookkeeper-operator`)